src/testing/burgers_stability.h

```diff
@@ -8,26 +8,23 @@
 namespace PHiLiP {
 namespace Tests {
 
-/// Burgers sine wave to shock.
-/** Ensure that the kinetic energy is bounded.
- *  Gassner 2017.
- */
 template <int dim, int nstate>
+/// Burgers' periodic unsteady test
 class BurgersEnergyStability: public TestsBase
 {
 public:
-    /// Constructor.
- BurgersEnergyStability(const Parameters::AllParameters *const parameters_input);
-    /// Ensure that the kinetic energy is bounded.
-    /** If the kinetic energy increases about its initial value, then the test should fail.
-     *  Gassner 2017.
-     */
-    int run_test () const override;
+        /// Constructor
+	BurgersEnergyStability(const Parameters::AllParameters *const parameters_input);
+        /// Run the testcase
+        int run_test () const override;
 private:
-    /// Computes an integral of the kinetic energy (solution squared) in the entire domain.
-    /** Uses Inverse of inverse mass matrix (?) to evaluate integral of u^2.
-     */
- double compute_energy(std::shared_ptr < PHiLiP::DGBase<dim, double> > &dg) const;
+    /// Function computes the energy
+	double compute_energy(std::shared_ptr < PHiLiP::DGBase<dim, double> > &dg) const;
```

Comment:
this function and `compute_conservation` is showing up in a lot of the tests you added, if they're all the same or similar, consider creating an intermediate class between `TestBase` and the relevant TestBase derived classes and move them there to reduce code repetition

Reply:
fixed as per issue #127

Reply:
resolving as it doesn't affect the core of the code